libs/product/README.md

```diff
@@ -9,3 +9,58 @@ provides clear interfaces, models, and factories for the frontend of an ecommerc
 ```
 npm install @daffodil/product
 ```
+
```

Comment:
I would move this into a separate "stories" folder.

Reply:
To clarify, I don't think this should be in the README of `product`. I think we should have a separate `stories` folder for docs which describes a "story" a developer would undergo.

Reply:
done